src/front/app/components/Challenges/Challenge.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Challenge = ({ challenge = {} }) => (
```

Comment:
you should specify each used property from challenge to component props to remove component dependency on model